homeassistant/components/media_player/volumio.py

```diff
@@ -210,3 +238,21 @@ def async_mute_volume(self, mute):
 
         return self.send_volumio_msg(
             'commands', params={'cmd': 'volume', 'volume': self._lastvol})
+
+    def async_select_source(self, source):
+        """Choose a different available playlist and play it."""
+        self._currentplaylist = source
+        return self.send_volumio_msg(
+            'commands', params={'cmd': 'playplaylist', 'name': source})
+
+    def async_clear_playlist(self):
+        """Clear players playlist."""
+        # FIXME
```

Comment:
What is needed to be fixed?

Reply:
Good question !
This clear function doesn't work with the Volumio rest API.
So, I just created a new PR for that: https://github.com/home-assistant/home-assistant/pull/12173